examples/svelte/.storybook/main.ts

```diff
@@ -1,6 +1,7 @@
-const preprocess = require('svelte-preprocess');
+import { withOverview } from 'bookcase-builder';
+import preprocess from 'svelte-preprocess';
 
-module.exports = {
+export default withOverview(__dirname)({
```

Comment:
Did these need to change to TS files, or is that just for convenience?

Reply:
For good type hints, of course, you can use `.js`, and bookcase-builder supports cjs.

Reply:
OK, normally I would want somewhat-unrelated changes like that to be in a different PR.  This PR should really just be the smallest number of changes to make the composition work correctly.  And there are problems with using ts config files in some cases in storybook (e.g. when `"type": "module"` is set), so I'm not sure if it will confuse others if they try to use these examples as a base for their own projects.  But, we can try this, and change it later if there is confusion.

Reply:
Ok, I learned. 👍